smart-contracts/contracts/BondingCurveHook.sol

```diff
@@ -82,6 +82,8 @@ contract BondingCurveHook is ILockKeyPurchaseHookV7 {
   ) external
   {
     require(msg.sender == lockAddress, 'UNAUTHORIZED_ACCESS');
+    address author = bytesToAddress(data);
+    require(msg.sender != author, 'MINT_TO_SELF');
```

Comment:
I think we should remove this. It's so easy to cheat anyway... and in the end, if someone pays the full price of a membership, I guess they can get the share?

Reply:
Yes, agreed. This is removed in the most current version (I haven't pushed it yet).

Reply:
fixed.